README.md

```diff
@@ -17,8 +17,8 @@ Creates Alma-compliant XML patron profiles from information extracted from the D
 - `DATA_WAREHOUSE_HOST` = The host for the Data Warehouse database.
 - `DATA_WAREHOUSE_PORT` = The port for the Data Warehouse database.
 - `DATA_WAREHOUSE_SID` = The system identifier for the Data Warehouse database instance.
+- `LOG_LEVEL` = The log level for the `alma-patronload` application
 - `S3_BUCKET_NAME` = The S3 bucket in which files are deposited.
 - `S3_PATH` = The path to the S3 bucket in which files are deposited.
```

Comment:
This is really the file path prefix for files deposited to the S3 bucket, right?